src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php

```diff
@@ -416,7 +416,40 @@ private function getExpressionResolvers(): array
 								new Name($classNames[0])
 							);
 						}
-						return null;
+
+						if ($classType->isClassStringType()->yes()) {
+							$templateClasses = $classType->getReferencedClasses();
+
+							if (count($templateClasses) === 1) {
```

Comment:
Why special-case an array with a single item? The code would work even if we removed this if, right?